neo4j/_async/driver.py

```diff
@@ -343,13 +407,16 @@ def session(
             fetch_size: int = ...,
             impersonated_user: t.Optional[str] = ...,
             bookmarks: t.Union[t.Iterable[str], Bookmarks, None] = ...,
+            ignore_bookmark_manager: bool = ...,
             default_access_mode: str = ...,
 
             # undocumented/unsupported options
             # they may be change or removed any time without prior notice
             initial_retry_delay: float = ...,
             retry_delay_multiplier: float = ...,
             retry_delay_jitter_factor: float = ...,
+            bookmark_manager: t.Union[AsyncBookmarkManager,
```

Comment:
why is the bookmark manager exposed here and not at the driver level?

Reply:
ah, maybe it's not the public API to create a session from the driver, or is it?

Reply:
This is the config option (i.e., `neo4j.AsyncGraphDatabase.driver(..., bookmark_manager=...)`). This can only go here or at the session level and the ADR specs it to go here.

But you are still right, that it's `neo4j.AsyncGraphDatabase.bookmark_manager(**some_config)` and `neo4j.GraphDatabase.bookmark_manager(...)` whereas it's `neo4j.bookmark_manager` in JS and Go, and porbably others. But since this methods needs to be able to return an async or sync BMM, depending on what driver you want to configure, I can't put it at the top level, unless I want to go `neo4j.bookmark_manager(...)` and `neo4j.async_bookmark_manager(...)`. Which would've been another valid option, I suppose. I'm indecisive as of which option would be better. My initial thoughts were to put it next to the driver factory method as it's functionality is pretty much coupled to the driver.